src/env-inl.h

```diff
@@ -916,6 +916,34 @@ inline void Environment::RemoveHeapSnapshotNearHeapLimitCallback(
                                         heap_limit);
 }
 
+inline v8::CpuProfilingResult Environment::StartCpuProfile(
+    v8::Local<v8::String> title) {
+  if (!cpu_profiler_) {
+    cpu_profiler_ = v8::CpuProfiler::New(isolate_);
+  }
+  v8::CpuProfilingResult result = cpu_profiler_->Start(title, true);
+  if (result.status == v8::CpuProfilingStatus::kStarted) {
+    node::Utf8Value key(isolate_, title);
+    pending_profiles_.insert({*key, result.id});
```

Comment:
```suggestion
    pending_profiles_.insert({key.ToString(), result.id});
```

We should probably even just straight up deprecate `operator*` here or make it return `std::string_view`, it's too easy to accidentally use the variants that return a `const char*` (which a) don't handle null bytes correctly and b) always require an extra string scan to look up the length)